src/GuideAtom.tsx

```diff
@@ -1,33 +1,224 @@
 import React from 'react';
 
 import { GuideAtomType } from './types';
+import { css, cx } from 'emotion';
+import { body, textSans } from '@guardian/src-foundations/typography';
+import { neutral } from '@guardian/src-foundations/palette';
+import { SvgInfo } from '@guardian/src-icons';
+import { Footer } from './components/Footer';
+import { Summary } from './components/Summary';
+import { GetPillarColour400, GetPillarColour300 } from './lib/PillarColours';
 
-const Body = ({ html }: { html: string }) => (
-    <div
-        dangerouslySetInnerHTML={{
-            __html: html,
-        }}
-    />
+const containerStyling = css`
+    display: block;
+    position: relative;
+    margin-bottom: 0.75rem;
+    margin-top: 1rem;
+`;
+
+const detailStyling = css`
+    margin: 16px 0 36px;
+    background: ${neutral[93]};
+    padding: 0 5px 6px;
+    border-image: repeating-linear-gradient(
+            to bottom,
+            ${neutral[86]},
+            ${neutral[86]} 1px,
+            transparent 1px,
+            transparent 4px
+        )
+        13;
+    border-top: 13px solid black;
+    position: relative;
+    summary {
+        list-style: none;
+        margin: 0 0 16px;
+    }
+
+    summary::-webkit-details-marker {
+        display: none;
+    }
+
+    summary:focus {
+        outline: none;
+    }
+`;
+
+const bodyStyling = css`
+    ${body.medium()}
+    p {
+        margin-bottom: 0.5rem;
+    }
+
+    ol {
+        list-style: decimal;
+        list-style-position: inside;
+        margin-bottom: 1rem;
+    }
+
+    ul {
+        list-style: none;
+        margin: 0 0 0.75rem;
+        padding: 0;
+        margin-bottom: 1rem;
+    }
+
+    ul li {
+        margin-bottom: 0.375rem;
+        padding-left: 1.25rem;
+    }
+
+    ul li:before {
+        display: inline-block;
+        content: '';
+        border-radius: 0.375rem;
+        height: 0.75rem;
+        width: 0.75rem;
+        margin-right: 0.5rem;
+        background-color: ${neutral[86]};
+        margin-left: -1.25rem;
+    }
+
+    b {
+        font-weight: bold;
+    }
```

Comment:
Is this needed? Does the user agent stylesheet get overriden in DCR?

Reply:
Yeah, bold styling gets overridden without this.

Reply:
I think you likely want to use the font weight here https://theguardian.design/2a1e5182b/p/930d69-overview/t/288fd4 so - 700? Otherwise you might get faux bold rather than use the actual bold typeface.